data/json/items/chemicals_and_resources.json

```diff
@@ -583,5 +583,32 @@
     "bashing": 0,
     "cutting": 0,
     "to_hit": -2
+    },
+    {
+    "type" : "AMMO",
+    "id" : "iodine_solution",
+    "name" : "iodine solution",
+    "weight" : 51,
+    "color" : "light_cyan",
+    "addiction_type" : "none",
```

Comment:
This should lose all the consumable-related fields: addiction_type, stim, quench etc.